lib/default_environment.js

```diff
@@ -0,0 +1,20 @@
+/* jshint browser: true */
+/* global chrome */
+
+function defaultEnvironment() {
+  if (typeof(window) !== 'undefined') {
+    // Browser
+    return { window: window };
+  } else if (typeof(chrome) !== 'undefined') {
+    // Chrome Extension
+    return { chrome: chrome };
```

Comment:
note these are not mutually exclusive - chrome extensions often have `window` and `chrome` defined, for example. I think maybe the default environment isn't the place to be opinionated about what gets selected from the globals, but rather just try to pass them along if they exist, e.g.:

    var env = {};
    if (typeof(window) !== 'undefined') {
      env.window = window;
    }
    if (typeof(chrome) !== 'undefined') {
      ...